src/main/java/htmlpublisher/HtmlPublisherTarget.java

```diff
@@ -115,14 +117,31 @@ public String getReportTitles() {
      * @since 1.4
      */
     @DataBoundConstructor
-    public HtmlPublisherTarget(String reportName, String reportDir, String reportFiles, String reportTitles, boolean keepAll, boolean alwaysLinkToLastBuild, boolean allowMissing) {
+    public HtmlPublisherTarget(String reportName, String reportDir, String reportFiles, String reportTitles, boolean keepAll, boolean alwaysLinkToLastBuild, boolean allowMissing, boolean escapeUnderscores) {
```

Comment:
Should be added in a `@DataboundSetter`.

Reply:
Thanks - didn't really know much about `@DataboundSetter` and how it fit into things but now changed to use and a great thing to know for future changes.

Reply:
The benefit is that for use in pipeline (unsure if relevant) those are considered optional.

https://jenkins.io/doc/developer/plugin-development/pipeline-integration/#constructor-vs-setters

I think configuration-as-code works similarly, but unsure. @ndeloof might know.